formats/cbor/commonMain/src/kotlinx/serialization/cbor/internal/Encoding.kt

```diff
@@ -32,6 +32,9 @@ private const val HEADER_NEGATIVE: Byte = 0b001_00000
 private const val HEADER_ARRAY: Int = 0b100_00000
 private const val HEADER_MAP: Int = 0b101_00000
 
+/** Value to represent an indefinite length CBOR item within a "length stack". */
+private const val LENGTH_STACK_INDEFINITE = -1
```

Comment:
Created constant to avoid confusion with `-1` returned by `readByte` (which is used in `skipElement`).